test/unittests/libethashseal/EthashTest.cpp

```diff
@@ -181,7 +181,7 @@ TEST(Ethash, etashVerify)
     header.setLogBloom({});
     header.setDifficulty(h256{131072});
     header.setNumber(1);
-    header.setGasLimit(3141562);
+    header.setGasLimit(8000000);
```

Comment:
Updating this field in the test requires the hashes to be updated too. Though I do not think it is important to change the gas limit in this test.

Reply:
@axic Maybe aleth maintainers can?

Reply:
I am not sure what you mean, but if you remove this part of your commit, it has a better chance of merging.

Reply:
@axic reverted, thanks 👍